src/naksu/mebroutines/remove/remove.go

```diff
@@ -0,0 +1,59 @@
+package remove
+
+import (
+  "fmt"
+  "errors"
+	"naksu/mebroutines"
+	"naksu/progress"
+)
+
+// Server removes all directories related to Vagrant and VirtualBox
+func Server() error {
```

Comment:
I'd name this `RemoveServer()` since it would make the context clearer

Reply:
lint does not like your suggestion: `func name will be used as remove.RemoveServer by other packages, and that stutters; consider calling this Server`